docs/Explorer.md

```diff
@@ -32,6 +32,14 @@ Multiple delete is not supported for the moment. See [issue 107](https://github.
 
 To delete a consumer group, it first must be stopped, else the `Delete` action will report an error.
 
+## Discover new cluster providers
+
+You can search for extensions contributing cluster providers in the extension gallery, by clicking on the `Discover Cluster Providers` button (also available via the command palette):
+
+![Discover Cluster Providers](assets/kafka-explorer-discover-providers.png)
+
+Those extensions must be labeled `kafka-provider`.
+

```

Comment:
Perhaps it should use tagged instead of labeled and add a package.json sample with this tag.

Reply:
Do you think you could add a screenshot here with the result of click on `Discover Cluster Providers` icon.

Reply:
not until there are providers available

Reply:
> Perhaps it should use `tagged `instead of `labeled `and add a package.json sample with this tag.

fixed